jina/serve/runtimes/gateway/http_fastapi_app_docarrayv2.py

```diff
@@ -258,6 +258,29 @@ async def streaming_get(request: Request):
             async def event_generator():
                 async for doc, error in streamer.stream_doc(
                     doc=input_doc_model(**query_params), exec_endpoint=endpoint_path
+                ):
+                    if error:
+                        raise HTTPException(status_code=499, detail=str(error))
```

Comment:
can we get this 499 from the proper fastapi status constants?

Reply:
I can't seem to find a fastapi constant for this, can you find me an example? This code is copied from the existing get endpoint.